### ISSUE TITLE
"ERROR; Cannot uninstall certifi" on certain platforms/versions (Linux and macOS, Python 3.6)

### ISSUE THREAD

**Issue by lbianchi-lbl** [CCSI-Toolset/FOQUS#974]
```sh
    Attempting uninstall: certifi
      Found existing installation: certifi 2016.9.26
  ERROR: Cannot uninstall 'certifi'. It is a distutils installed project and thus we cannot accurately determine which files belong to it which would lead to only a partial uninstall.
```

This has caused partial failures in our nightly builds starting sometime this week. At the moment, only the following combinations seem to be affected:

- macOS, Python 3.6
- Linux (Ubuntu), Python 3.6

Possible causes can be:

- Recent updates to `pip`
- Recent platform-dependent updates to `certifi`
- Recent changes in the installed software of the GitHub Actions runners
- ... and any combination of the former

**Comment by lbianchi-lbl** [comment]
PR #984 addresses this issue. The workaround of adding the `--ignore-installed` flag to the `pip install` command seems to be working, e.g.:

```sh
pip install -r requirements-dev.txt --ignore-installed
```

**Comment by lbianchi-lbl** [comment]
Following @boverhof's pointer that `certifi` was updated on Oct 8th, which seems to be confirmed by looking at when the builds started failing on Oct 9th:

![image](https://user-images.githubusercontent.com/48035537/144130362-80854ee4-b40b-4afd-87b8-8d440fe998d7.png)

**Comment by ksbeattie** [comment]
Addressed in #984
